app/javascript/components/service-request-default/service-request-default.schema.js

```diff
@@ -1,13 +1,16 @@
 import { componentTypes } from '@@ddf';
 
 const findUser = (searchHere) => {
-  if (searchHere.users.length > 1) {
-    return [{
-      label: searchHere.selectedUser.charAt(0).toUpperCase() + searchHere.selectedUser.slice(1),
-      value: searchHere.selectedUser,
-    }];
-  } if (searchHere.users.length === 1) {
-    return [searchHere.users[0]];
+  if (searchHere.users.length >= 1) {
+    const obj = [];
```

Comment:
we can put `let obj = [{ label: __('All'), value: 'all' }];` to make it the first option.

Reply:
It wouldn't stay first once we run `obj.sort((a, b) => a.label - b.label);` if theres a user like `aaaa`